middleware/mqtt_handler.py

```
@@ -149,6 +144,11 @@ def _handle_uid_only_tag(client: mqtt.Client, scanner_cfg: dict, uid: str, topic
         logger.warning(f"Low spool: {name} ({remaining:.1f}g) on {target_id}")
 
 
+    current = app_state.active_spools.get(target)
+    if current == spool_id:
+        logger.info(f"Ignoring same spool {spool_id} on {target}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)